### ISSUE TITLE
No increase in code coverage

### ISSUE THREAD

**Issue by jamal-ansary** [plasma-umass/coverup#8]
Hello, 
The model doesn't save the test units as a .py file when there is no increase in code coverage. it saves them as log file. is there an option to override that?

**Comment by jaltmayerpizzorno** [comment]
No, currently there isn't, but it would be relatively easy to add. I'm curious, though: how that would be useful?

**Comment by jamal-ansary** [comment]
i had a python code with bug in it. i also had a unit test that didn't reveal the bug. the coverage didn't increase after prompting with chat gpt, so the test cases weren't saved. when i was reading log file, one of the test cases would be able to detect the bug. Just something that i came across. 
Any way, propos for the great work.

**Comment by jaltmayerpizzorno** [comment]
Thank you :)

I am not quite clear on the situation, though... was the test dropped because it (exposed the bug and) failed, or because it didn't increase coverage?

**Comment by jamal-ansary** [comment]
because it didn't increase coverage.
